### DISCUSSION TITLE
Make QR code exportable as image

### DISCUSSION THREAD

**Discussion by Resaki1** [inovex/scrumlr.io#3870] [Proposals]
Currently, the QR code to share a session is rendered as an SVG. Right-clicking it does not give the option to save the QR code as an image (at least in Chromium). Of course you could make a screenshot, but rendering it as an image would make it easier imo

**Comment by brandstetterm** [comment]
I really like your suggestion and I think we should do it! 
At first I was wondering how this would be useful and why you would want to download the QR code. But after thinking about it for a while, I came to the conclusion that you might want to embed the QR code into something like a picture, poster, etc.
